controllers/provider-gcp/pkg/internal/infrastructure/terraform.go

```diff
@@ -117,10 +133,16 @@ func ComputeTerraformerChartValues(
 		"vpc":         vpc,
 		"clusterName": infra.Namespace,
 		"networks": map[string]interface{}{
-			"pods":     extensionscontroller.GetPodNetwork(cluster),
-			"services": extensionscontroller.GetServiceNetwork(cluster),
-			"workers":  workersCIDR,
-			"internal": config.Networks.Internal,
+			"pods":              extensionscontroller.GetPodNetwork(cluster),
+			"services":          extensionscontroller.GetServiceNetwork(cluster),
+			"workers":           workersCIDR,
+			"internal":          config.Networks.Internal,
+			"enableVPCFlowLogs": enableVPCFlowLogs,
+			"flowLog": map[string]interface{}{
```

Comment:
```suggestion
			"flowLogs": map[string]interface{}{
```

Can you please only provide this to the Helm chart if the `InfrastructureConfig` contains config for the flow logs? If not, simply omit/don't pass it.